tagging/test/test_tag_service.py

```diff
@@ -50,11 +51,16 @@ def test_unique_uid_tag_set(tag_svc: TagService):
         tag_svc.create_tagging_event(another_event)
 
     with pytest.raises(DuplicateKeyError):
```

Comment:
"create_dataset" will not raise this error by itself anymore because it assigns a new UID every time it is called. I added some lines to force the duplicate directly in mongo, but I was thinking about removing this test entirely.
Let me know what you think.